h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -723,6 +723,22 @@ public void updateAuxTreeWeights(Frame frame, String weightsColumn) {
             .getNodeWeights();
     AuxNodeWeights auxNodeWeights = new AuxNodeWeights(model_info().getAuxNodeWeightsKey(), nodeWeights);
     DKV.put(auxNodeWeights);
+
+    UpdateAuxTreeWeightsReport report = new UpdateAuxTreeWeightsReport();
+    report._warn_classes = new int[0];
+    report._warn_trees = new int[0];
+    for (int treeId = 0; treeId < nodeWeights.length; treeId++) {
+      if (nodeWeights[treeId] == null)
+        continue;
+      for (double w : nodeWeights[treeId]) {
+        if (w == 0) {
+          report._warn_trees = ArrayUtils.append(report._warn_trees, treeId);
+          report._warn_classes = ArrayUtils.append(report._warn_classes, 0);
```

Comment:
Why we are appending 0?

Reply:
class 0, multinomial is not supported but could be added in the future without API change